dataplane/api-server/src/server.rs

```diff
@@ -99,16 +99,11 @@ impl Backends for BackendService {
         let ip = pod.ip;
         let ip_addr = std::net::Ipv4Addr::from(ip);
 
-        let device = match if_name_for_routing_ip(ip_addr) {
+        let ifindex = match if_index_for_routing_ip(ip_addr) {
             Ok(device) => device,
```

Comment:
For consistency, perhaps?

```suggestion
            Ok(ifindex) => ifindex,
```